code/controllers/subsystems/job.dm

```diff
@@ -507,6 +508,9 @@
 		else
 			SSjobs.odin_despawn_mob(src) //somehow they can't spawn at cryo, so this is the only recourse of action.
 
+/mob/living/carbon/human/proc/remove_newplayer_status(mob/living/carbon/human/H)
```

Comment:
The argument should not be used. It should affect `src.newarrival = 0` `H` is not necessary.